xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/xwiki.js

```diff
@@ -668,33 +577,6 @@ Object.extend(XWiki, {
     }.bind(this));
   },
 
-  /**
-   * Extracts the file name from the value of the specified file input.
-   */
-  extractFileName: function(fileInput) {
```

Comment:
I didn't move that one but I entirely removed it as it's apparently not used anywhere in XS and I  thought it wasn't exposed but maybe I'm wrong?

Reply:
It's definitely exposed, you can check by running `XWiki.extractFileName` in the debug console of any XWiki instance.
Removing this is a breaking change.

Reply:
So now do we want to deprecate it or not?

Reply:
As for any deprecation, we need to have a discussion on the forum.

Reply:
Done: https://forum.xwiki.org/t/deprecate-xwiki-extractfilename-from-xwiki-js/14957/7